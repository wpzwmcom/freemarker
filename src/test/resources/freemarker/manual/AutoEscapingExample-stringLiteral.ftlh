<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->
<#assign s = "Foo & bar">
${s}
${"${s} & baz"}