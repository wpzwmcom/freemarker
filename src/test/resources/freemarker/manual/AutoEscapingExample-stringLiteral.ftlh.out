Foo &amp; bar
Foo &amp; bar &amp; baz